docs/examples/viz_slicer.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from fury import actor, window
+from fury.utils import get_slices, show_slices
+from dipy.data import read_mni_template
+
+###############################################################################
+# Let's read the 3D data from dipy
+
+nifti = read_mni_template()
+data = np.asarray(nifti.dataobj)
+print(data.shape)
+
+###############################################################################
+# Create slice actor to visualize the 3D data as XY, YZ and XZ slices.
+
+slicer_actor = actor.slicer(data)
+scene = window.Scene()
+scene.add(slicer_actor)
+
+
+def handle_pick_event(event):
+    info = event.pick_info
+    intensity = np.asarray(info["rgba"].rgb).mean()
+    print(f"Voxel {info['index']}: {intensity:.2f}")
+
+
+def handle_wheel_event(event):
```

Comment:
Does it means wheel will not work if we do not add that ?

Reply:
Also, it does not work well, the camera is scrolling as the same time of the slice. it is confusing.

Reply:
Yes this is not the default behavior out of the box I created to confirm that we can scroll through slices.

I can update the tutorial to handle it with arrow keys or I can add this out of the box in the actor.

I refrain from adding to the actor as it limits the customizability.